scss/components/_side-nav.scss

```diff
@@ -30,6 +30,15 @@
   &:last-child {
     padding-bottom: 0;
   }
+
+  &.is-disabled {
```

Comment:
Ah! My bad. I was looking at a similar but different pattern:
![image](https://cloud.githubusercontent.com/assets/1696495/14445777/d0a6aef2-0004-11e6-8524-a4c1ec70fd85.png)

I think what you have is the right way to go.